File: year2020/month09/day0908/ms_0908.cpp
#include <iostream>

using namespace std;

void changeValue(int* pointer);

int main() {
	// data[0] 이 작은 수, data[1] 이 큰 수를 가진다.
	int data[2];

	for (int i = 0; i < 2; i++)
		scanf_s("%d", &data[i]);

	if (data[0] > data[1]) {
		int temp = data[0];
		data[0] = data[1];
		data[1] = temp;
	}

	changeValue(data);
	cout << data[0] << ' ' << data[1] << endl;

	return 0;
}

void changeValue(int* pointer) {
	*pointer *= 2;
	*(++pointer) /= 2;
}

File: year2020/month09/day0908/ms_0908.cs
using System;

namespace algorithm_52
{
    class ms_0908
    {
        static void changeValue(ref int mult, ref int div) 
        {
                mult *= 2;
                div /= 2;
        }

        static void Main(string[] args)
        {
            // x 가 작은 수, y 가 큰 수를 저장
            string[] input = Console.ReadLine().Split(' ');
            int x = Convert.ToInt32(input[0]);
            int y = Convert.ToInt32(input[1]);

            if(x > y) 
            {
                int temp = x;
                x = y;
                y = temp;
            }

            changeValue(ref x, ref y);
            Console.WriteLine($"{x} {y}");
        }
    }
}
